controllers/sr_controller/sr/robot/motor.py

```diff
@@ -28,33 +28,28 @@ def translate(sr_speed_val, sr_motor):
     return map_to_range(
         -SPEED_MAX,
         SPEED_MAX,
-        -sr_motor.max_speed,
-        sr_motor.max_speed,
+        -sr_motor._max_speed,
+        sr_motor._max_speed,
         sr_speed_val,
     )
 
 
-class Motor(object):
+class MotorBoard(object):
```

Comment:
Renaming this class seems unrelated? Is this to match the "real" SR API or just for clarity? Either way, maybe extract this change to simplify this PR?

Reply:
I think it's for clarity. The original matches the SR implementation